xml/telnet_banners.xml

```diff
@@ -816,4 +816,589 @@
     <param pos="4" name="os.version"/>
     <param pos="5" name="os.version.version"/>
   </fingerprint>
+  <fingerprint pattern="^Red Hat Linux release ([^\\s]+)\\s*.*$">
+    <description>RedHat general purpose linux</description>
+    <example _encoding="base64" os.version="9 (Shrike)">
+      UmVkIEhhdCBMaW51eCByZWxlYXNlIDkgKFNocmlrZSlcbktlcm5lbCAyLjQuMjAtOCBvbiBhbiBpNjg2XG5sb2dpbjo=
+   </example>
+    <param pos="0" name="hw.vendor" value="RedHat"/>
+    <param pos="0" name="he.family" value="Linux"/>
+    <param pos="0" name="hw.device" value="Linux"/>
+    <param pos="1" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="Red Hat Enterprise Linux ES release (.*) \(.*\).*Kernel (.*) on a[^ ]* ([^ ]*\d)" flags="REG_MULTILINE">
+    <description>RedHat Enterprise Linux ES</description>
+    <example _encoding="base64" os.version="3" linux.kernel.version="2.4.21-47.EL" os.arch="x86_64">
+      UmVkIEhhdCBFbnRlcnByaXNlIExpbnV4IEVTIHJlbGVhc2UgMyAoVGFyb29uIFVwZGF0ZSA5KQpLZXJuZWwgMi40LjIxLTQ3Lk
```

Comment:
Please include the plain-text version of the example in a comment above each example, here and everywhere.  Otherwise, review and maintenance of these files will become harder.